File: app/src/main/java/com/psliusar/layers/sample/LayerFragmentActivity.java
package com.psliusar.layers.sample;

import android.os.Bundle;

import com.psliusar.layers.sample.screen.fragment.FragmentLayer;

import androidx.fragment.app.FragmentActivity;

public class LayerFragmentActivity extends FragmentActivity {

    @Override
    protected void onCreate(Bundle state) {
        super.onCreate(state);
        setContentView(R.layout.activity_main);
        if (state == null) {
            getSupportFragmentManager().beginTransaction().add(R.id.container, new FragmentLayer(), null).commit();
        }
    }
}

File: app/src/main/java/com/psliusar/layers/sample/LayerFragmentActivity.kt
package com.psliusar.layers.sample

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import com.psliusar.layers.sample.screen.fragment.FragmentLayer

class LayerFragmentActivity : AppCompatActivity(R.layout.activity_main) {

    override fun onCreate(state: Bundle?) {
        super.onCreate(state)
        val toolbar = findViewById<Toolbar>(R.id.toolbar)
        setSupportActionBar(toolbar)
        toolbar.setNavigationOnClickListener {
            onBackPressed()
        }
        supportActionBar?.setDisplayHomeAsUpEnabled(true)

        if (state == null) {
            supportFragmentManager.beginTransaction().add(R.id.container, FragmentLayer(), null).commit()
        }
    }
}
